lib/utilities/pristine.js

```diff
@@ -178,106 +200,100 @@ function copyUnderTestApp(pristineAppPath, underTestAppPath) {
 }
 
 function addEmberToBowerJSON(appName, version) {
```

Comment:
I changed the `removeEmberSourceFromDependencies` to be `updateEmberSource` which will now:

- If using Bower, always remove `ember-source`.
- If not using Bower, updates the `ember-source` version to the specified `emberVersion` (defaulting to `beta`).

Does this seem reasonable? It'll be a slight change, but after looking at engines and fastboot tests, I don't think it should affect either. Regardless, we'll want to do a minor version bump on the next release (instead of a patch release).